public/pages/Dashboard/Components/utils/SideBar.tsx

```diff
@@ -0,0 +1,59 @@
+import { APP_PATH } from '../../../../utils/constants';
+import React, { Component } from 'react';
+import { EuiSideNav, EuiPageSideBar } from '@elastic/eui';
+
+type SideBarState = {
+  selectedItemName: string;
+};
+
+export class SideBar extends Component<{}, SideBarState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      selectedItemName: 'Lion stuff',
```

Comment:
Typo ?

Reply:
Not a typo.  Just initialize the selected item to some name.  When a user selects a link, this field would be assigned to another value.